code/game/machinery/iv_drip.dm

```diff
@@ -7,11 +7,14 @@
 	var/mob/living/carbon/attached = null
 	var/mode = 1 // 1 is injecting, 0 is taking blood.
 	var/obj/item/weapon/reagent_containers/beaker = null
+	var/list/drip_containers = list(/obj/item/weapon/reagent_containers/blood,
+											/obj/item/weapon/reagent_containers/food/drinks,
+											/obj/item/weapon/reagent_containers/glass)
```

Comment:
Just so everyone is aware the rag is a subtype of this type shall I make a blacklist specifically against the rag?
Also does anyone know of any of the drinks subtype that I should blacklist if needed?